src/main/resources/db/changelog/changes/v1.1.0/update-drv-item-details-definition.xml

```diff
@@ -239,14 +239,14 @@
             },
             {
               "name": "item_material_type",
-              "source": {
-                "columnName": "material_type_name",
-                "entityTypeId": "917ea5c8-cafe-4fa6-a942-e2388a88c6f6"
-              },
               "dataType": {
                 "dataType": "stringType"
               },
-              "idColumnName": "item_material_type_id",
+              "valueSourceApi": {
+                "path": "material-types",
+                "valueJsonPath": "$.mtypes.*.id",
+                "labelJsonPath": "$.mtypes.*.name"
```

Comment:
Alternatively, this could jave been
```json
                "valueJsonPath": "$..id",
                "labelJsonPath": "$..name"
```
I opted for the more targetted json path, just to be safe